variegated-hal/src/machine_mechanism/single_boiler_mechanism.rs

```diff
@@ -70,7 +69,6 @@ impl<'a> SingleBoilerBrewMechanism<'a> {
     }
 }
 
-#[async_trait]
```

Comment:
This doesn't work. The `async_trait` crate is still needed, because the traits are otherwise not dyn-compatible.